src/OrchardCore.Modules/OrchardCore.Email/Views/Admin/Index.cshtml

```diff
@@ -8,31 +8,31 @@
     <div class="form-group">
         <fieldset class="form-group" asp-validation-class-for="To">
             <label asp-for="To">@T["To"] <span asp-validation-for="To"></span></label>
-            <input asp-for="To" class="form-control" type="text" />
+            <input asp-for="To" class="form-control" type="email" pattern="^[a-zA-Z0-9._+-]+&#64;[a-zA-Z0-9.-]+\.[a-zA-Z]{2,9}$" />
```

Comment:
Do we have to set a regular expression here? Isn't just `type=email` sufficient?

Reply:
I can test again but it seemed that it would allow for asd@asd and it would then fail.

Reply:
I am fine with that, my concern is that by adding too much validation we'll prevent valid emails from working.